src/internal/storage/chunk/reader.go

```diff
@@ -103,7 +105,8 @@ type DataReader struct {
 
 func newDataReader(ctx context.Context, s *Storage, client Client, dataRef *DataRef, offset int64) *DataReader {
 	return &DataReader{
-		ctx:      ctx,
+		ctx: pctx.Child(ctx, fmt.Sprintf("dataReader('%s')", Base64Hash(dataRef)),
```

Comment:
We should use the `pfs.EncodeHash` utility function if we want to record hashes. Also, it seems like the chunk id may be more valuable than the data reference hash since it won't be equal to the chunk hash when it doesn't reference the full chunk.